Sources/Purchasing/Purchases.swift

```diff
@@ -91,9 +91,10 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
     private let operationDispatcher: OperationDispatcher
 
     /**
-     * Enable automatic collection of Apple Search Ads attribution. Defaults to `false`.
+     * Enable automatic collection of AdServices attribution token. Defaults to `false`.
      */
-    @objc public static var automaticAppleSearchAdsAttributionCollection: Bool = false
+    @available(iOS 14.3, *)
```

Comment:
I think this will need the `tvOS`/`macCatalyst` (`macOS` too?) equivalents too

Reply:
yep good call. xcode never suggests the full checks for us, do they...

Reply:
Nope, only for the currently selected target unfortunately.